tabrepo/benchmark/models/wrapper/abstract_class.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+import numpy as np
+import pandas as pd
+from autogluon.core.data.label_cleaner import LabelCleaner, LabelCleanerDummy
+from autogluon.core.metrics import get_metric, Scorer
+from autogluon.features import AutoMLPipelineFeatureGenerator
+from autogluon_benchmark.utils.time_utils import Timer
```

Comment:
Can we avoid this? This would make it a strong requirements? (We have a timer class in TabRepo right?)

Reply:
Agreed, but for now I am keeping. As mentioned above, I will soon remove the dependency on AG-Bench and AG-Benchmark entirely.

Reply:
This will be its own dedicated PR